.github/workflows/ci.yml

```diff
@@ -9,12 +9,11 @@ jobs:
     - name: Check Whitespace
       run: git diff --check -- HEAD~1
   rspec:
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         ruby: [2.1, 2.3, 2.4, 2.5, 2.6, 2.7, jruby]
-        os: [ubuntu-latest]
```

Comment:
I actually recommend you add osx, but as a warning: OSX is *much* slower on github actions so I recomend to only test a subset of ruby releases on it to preserve personal sanity when iterating.

Reply:
@mbj You think this code could/would behave differently on OSX?

Reply:
@dkubb yes. As OSX and MRI have slightly different kernel thread primitives which will evenutally affect the GIL behavior I think for this gem the "ruby-platform" semantics may be broken.

Reply:
@mbj ok, that sounds good enough for me. I will remove this specific commit and adds OSX to the OS matrix.

Reply:
@mbj ok, you're right this is really slow. It slows down my feedback loop too much for regular development so I don't think I'm going to add it to the normal CI step.

However, I do plan on adding CI configuration in the future where I do both of these things:

- Do a full mutant run *without* `--since`
- Run all tests on all ruby versions
- Run all tests on ubuntu-latest and macOS-latest.
- Run every 24 hours

I plan on removing most of the non-3.0 ruby versions from the "regular" run and then have the full 24 hour run all sorts of combinations.